spring-cloud-contract-verifier/src/test/groovy/org/springframework/cloud/contract/verifier/util/ContentUtilsSpec.groovy

```diff
@@ -22,4 +23,11 @@ class ContentUtilsSpec extends Specification {
 		expect:
 			"test" == ContentUtils.GET_TEST_SIDE(dslProperty)
 	}
+
+    def "should return XmlSlurper with default error handler"() {
+        given:
+        XmlSlurper xmlSlurper = ContentUtils.getXmlSlurperWithDefaultErrorHandler()
+        expect:
+        xmlSlurper.getErrorHandler() in DefaultHandler
```

Comment:
`DefaultHandler` is not a collection. Use `instanceof` instead of `in`.

Reply:
In Groovy we can use isCase or with a help of Iintellij idea help it could be `in`.
If you want to make it very explicit I can use instanceof.

Reply:
Yes, that's more intuitive to everyone.

Reply:
done